simulator/src/main/resources/app.properties

```diff
@@ -1 +1 @@
-blockStream.folderRootPath=/Users/user/Projects/hedera-block-node/simulator/src/main/resources/block-0.0.3/
+
```

Comment:
The removal of the property is intentional, the property having the hardcoded value to my specific path on my computer since the previous PR was accidental 😅